chain-tx-enclave/mls/tests/verify_test_vectors.rs

```diff
@@ -0,0 +1,42 @@
+use rustls::internal::msgs::codec::Codec;
+
+use mls::{keypackage::DEFAULT_LIFE_TIME, KeyPackage, KeyPackageVerifyError};
+use ra_client::EnclaveCertVerifierConfig;
+
+const IAS_CERT: &[u8] =
+    include_bytes!("../../../client-core/src/cipher/AttestationReportSigningCACert.pem");
+
+#[test]
+fn verify_keypackage_test_vector_mock() {
+    // keypackage_mock.bin is generated by golang implementation
+    static VECTOR: &[u8] = include_bytes!("test_vectors/keypackage_mock.bin");
+    let kp = <KeyPackage>::read_bytes(VECTOR).expect("decode");
+    let cfg = EnclaveCertVerifierConfig {
+        signing_ca_cert_pem: IAS_CERT.into(),
+        valid_enclave_quote_statuses: vec!["OK".into(), "GROUP_OUT_OF_DATE".into()].into(),
```

Comment:
> `GROUP_OUT_OF_DATE` isn't a valid response

```
Your remote attestation proxy server will begin to receive the “GROUP_OUT_OF_DATE” response described above for attestation requests originating from Intel SGX enabled platforms whose security requirements no longer meet these newly updated requirements.
```
Is it because the env generating this certificate is not up-to-date?

Reply:
or should have comment / warning it's only for testing purposes

Reply:
Added a `FIXME` for now.